<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>CREAiT</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1366</width>
     <height>768</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>14</number>
    </property>
    <property name="leftMargin">
     <number>14</number>
    </property>
    <property name="topMargin">
     <number>14</number>
    </property>
    <property name="rightMargin">
     <number>14</number>
    </property>
    <property name="bottomMargin">
     <number>14</number>
    </property>
    <item>
     <widget class="QWidget" name="rightLayout" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="Header" name="header" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>42</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>42</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <widget class="HomePage" name="homePage">
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="ProjectsViewPage" name="projectsViewPage"/>
         <widget class="CreateProjectPage" name="createProjectPage"/>
         <widget class="QWidget" name="mainSection">
          <property name="styleSheet">
           <string notr="true">#mainSection {
	background-color: white;
	border: 1px solid #ccc;
	border-top-left-radius: 16px;
	border-top-right-radius: 16px;
}
</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QStackedWidget" name="mainPanel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="currentIndex">
              <number>2</number>
             </property>
             <widget class="ProjectPage" name="projectsPage">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="accessibleDescription">
               <string/>
              </property>
             </widget>
             <widget class="DocumentsPage" name="documentsPage"/>
             <widget class="ReviewsPage" name="reviewsPage"/>
             <widget class="AutomationsPage" name="automationsPage"/>
             <widget class="ProjectSettingsPage" name="projecSettingsPage"/>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>26</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="logoContainer" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>68</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="logo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../../../Resources/Resources.qrc">:/images/logo-lg.svg</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="SideBar" name="sideBar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>102</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SideBar</class>
   <extends>QWidget</extends>
   <header>sidebar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Header</class>
   <extends>QWidget</extends>
   <header>header.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AutomationsPage</class>
   <extends>QWidget</extends>
   <header>automationspage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProjectPage</class>
   <extends>QWidget</extends>
   <header>projectpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DocumentsPage</class>
   <extends>QWidget</extends>
   <header location="global">documentspage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HomePage</class>
   <extends>QWidget</extends>
   <header>homepage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProjectSettingsPage</class>
   <extends>QWidget</extends>
   <header>projectsettingspage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreateProjectPage</class>
   <extends>QWidget</extends>
   <header>createprojectpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReviewsPage</class>
   <extends>QWidget</extends>
   <header location="global">reviewspage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProjectsViewPage</class>
   <extends>QWidget</extends>
   <header location="global">projectsviewpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
